samples/KK.DotNet.BackgroundTasks.Scheduled.Sample.Web/BackgroundServices/RuntimeSampleTask.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace KK.DotNet.BackgroundTasks.Scheduled.Sample.Web.Backgroundservices
+{
+    using Microsoft.Extensions.Logging;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using KK.DotNet.BackgroundTasks.Scheduled;
+
+    public class RuntimeSampleTask : IScheduledTask
+    {
+        private readonly ILogger<RuntimeSampleTask> logger;
+
+        public RuntimeSampleTask(
+            IScheduledTaskOptions<RuntimeSampleTask> options,
+            ILogger<RuntimeSampleTask> logger
+        )
+        {
+            this.Options = options;
+            this.logger = logger;
+        }
+
+        public IScheduledTaskOptions<IScheduledTask> Options { get; }
+
+        public async Task ExecuteAsync(CancellationToken stoppingToken)
+        {
+            this.logger.LogDebug("Runtime Sample Task started");
+
+            _ = stoppingToken.Register(() => this.logger.LogDebug("Runtime Sample Task forced stopping."));
+
+            var runCount = 0;
+            while (runCount < 5)
```

Comment:
Personally, I'd prefer a for-loop, but i assume thats a matter of style.

Reply:
good point, I will change this.